mfr/extensions/utils.py

```diff
@@ -0,0 +1,22 @@
+from urllib.parse import parse_qs, urlencode
+
+from mfr.extensions import settings
+
+
+def get_local_export_url(urlparsed_url):
+    """
+    Replace the docker domain with localhost (for use in local development).
+
+    :param parsed_url: the default export URL which is parsed by `urllib.parse.urlparse`
+    :return: the tweaked export URL for local development
+    """
+
+    query_dict = parse_qs(urlparsed_url.query, keep_blank_values=True)
+    # TODO: why do we pop `mode`?
+    query_dict.pop('mode', None)
+    urlparsed_url = urlparsed_url._replace(query=urlencode(query_dict, doseq=True))
+    urlparsed_url = urlparsed_url._replace(
+        netloc="{}:{}".format(settings.LOCAL_HOST, urlparsed_url.port)
+    )
```

Comment:
I'm not familiar with the `_replace` method.  What does it do, and why are you using a private method?  Is there another way to do this?

Reply:
I'm sure this can be done many other ways, but I was trying to do the least amount of converting. This way I am just manipulating the ParseResult, a namedtuple subclass. Dropping the "mode" query parameter and switching the netloc.

https://docs.python.org/3.6/library/collections.html#collections.somenamedtuple._replace